modules/consensus/accept_bench_test.go

```diff
@@ -0,0 +1,73 @@
+package consensus
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/gateway"
+)
+
+// benchmarkEmptyBlocks is a benchmark that mines many blocks, and
+// measures how long it takes to add them to the consensusset
+func benchmarkEmptyBlocks(bt *testing.B) error {
+	bt.StopTimer()
+
+	// Create an alternate testing consensus set, which does not
+	// have any subscribers
+	testdir := build.TempDir(modules.ConsensusDir, "BenchmarkEmptyBlocksB")
+	g, err := gateway.New(":0", filepath.Join(testdir, modules.GatewayDir))
+	if err != nil {
+		return err
+	}
+	cs, err := New(g, filepath.Join(testdir, modules.ConsensusDir))
+	if err != nil {
+		return errors.New("Error creating consensus: " + err.Error())
+	}
+
+	// The test dir will be reset each time the benchmark
+	// is done.
+	cst, err := createConsensusSetTester("BenchmarkEmptyBlocks")
+	if err != nil {
+		return errors.New("Error creating tester: " + err.Error())
+	}
+	for _, bID := range cst.cs.currentPath[1:] {
+		err = cs.AcceptBlock(cst.cs.blockMap[bID].block)
+		if err != nil {
+			return err
+		}
+	}
+
+	for j := 0; j < bt.N; j++ {
```

Comment:
throw a `bt.ResetTimer()` right here

Also, idiomatic go typically has the benchmark variable called `b`, it should not be replaced by the block variable. You should call the `testing.B` `b`, and then just call blocks `block`. It's a bit more verbose but will be less confusing to golang programmers unfamiliar with the project.